storage/results.go

```diff
@@ -28,7 +28,7 @@ type ExecutionResults interface {
 	ForceIndex(blockID flow.Identifier, resultID flow.Identifier) error
 
 	// BatchIndex indexes an execution result by block ID in a given batch
```

Comment:
A short documentation what the behaviour of the `ForceIndex` flag is would be great. Thanks.